WordPress-VIP/ruleset.xml

```diff
@@ -14,6 +14,41 @@
 	<!-- https://vip.wordpress.com/documentation/code-review-what-we-look-for/#commented-out-code-debug-code-or-output -->
 	<rule ref="Squiz.PHP.CommentedOutCode" />
 
+	<!-- https://vip.wordpress.com/documentation/vip/code-review-what-we-look-for/#eval-and-create_function -->
+	<!-- https://vip.wordpress.com/documentation/vip/code-review-what-we-look-for/#settings-alteration -->
```

Comment:
This link no longer applies to this sniff. It should be moved down to the rule inclusion of `WordPress.PHP.DiscouragedPHPFunctions` / `WordPress.PHP.DiscouragedPHPFunctions.runtime_configuration`